### ISSUE TITLE
Report In the navigation bar  not Translated to Arabic

### ISSUE THREAD

**Issue by marammahfoudhi** [idurar/idurar-erp-crm#1069]
**Describe the bug**
When translating the application interface to Arabic, Report in the navigation bar remains in English.

**To Reproduce**

1- Log in to the application
2- Navigate to the translation settings on the upper right .
3-Change the language to Arabic.
4-Go to the report section.
5-Notice that the report remains in English 

**Expected behavior**
The report section should be translated into Arabic when the language is switched to Arabic.

**Screenshots**

![translation issue](https://github.com/idurar/idurar-erp-crm/assets/125041992/6fc8a1b5-97dc-4717-a3c9-a09e04e11885)

Desktop:
OS: [e.g., Windows 11]

**Comment by Houssemdaoud** [comment]
Is this problem worked on cause i want to work on it

**Comment by marammahfoudhi** [comment]
I've fixed the Arabic translation for the report section. However, there are still issues with translations into other languages.

**Comment by Houssemdaoud** [comment]
> I've fixed the Arabic translation for the report section. However, there are still issues with translations into other languages.

did u find the report button when you locally launch the application cause i couldn't find it.

**Comment by marammahfoudhi** [comment]
Yes i did find the report button when the application is launched locally,  Ensure that the "Report" section is added to the navigation bar component for all supported langauages

**Comment by marammahfoudhi** [comment]
Yes i did find the report button when the application is launched locally, Ensure that the "Report" section is added to the navigation bar component for all supported languages
